cclient/serialization/tests/test_json.h

```diff
@@ -84,6 +84,45 @@
 
 #define TEST_RAW_TRYTES2 "RAWTRYTESHASH99999999"
 
+#define WERE_ADDRESSES_SPENT_FROM_TRYTES                                       \
```

Comment:
I think hashes and transaction's trytes in most cases can be shared between commands, so could you please rename it to TRANSACTION_TRYTES and see that other tests passes with that trytes?